Updated to use Billing Library 7.0.0

kanetik opened the pull request:
Did what is needed to make MBE work with the latest version of Google Play Billing Library ([Migration Guide](https://developer.android.com/google/play/billing/migrate-gpblv7))

Also updated most libraries and fixed build errors and lint suggestions.

kanetik committed:
updated dependencies

kanetik committed:
build issues

kanetik committed:
Updated Billing Library

kanetik committed:
couple more build errors

kanetik committed:
billing library update

kanetik committed:
couple more updates

kanetik committed:
bye jetifier

kanetik committed:
cleanup

kanetik committed:
don't let connection attempts continue forever

michal-luszczuk commented:
Hello 👋 
Thank you for your posting this changes.
I will decline this PR as i just updated the code even with the newer versions, and also introduced more changes.

The new version with 7.0.0 support will be released soon

kanetik commented:
FYI - In recent months, I'd forked your repo and make changes for myself.
One of the biggest changes was to retry logic, trying to adhere to the
guidance Google has provided here:
https://developer.android.com/google/play/billing/errors.

I don't love how I've implemented parts of it, but it seems to work, and
anecdotally, my completed purchases have increased since I did that. If you
care to see what I did, feel free to check it out here:
https://github.com/kanetik/MakeBillingEasy

Thanks!

On Sun, Jul 21, 2024 at 11:36 AM Michał ***@***.***> wrote:

> Hello 👋
> Thank you for your posting this changes.
> I will decline this PR as i just updated the code even with the newer
> versions, and also introduced more changes.
>
> The new version with 7.0.0 support will be released soon
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/michal-luszczuk/MakeBillingEasy/pull/8#issuecomment-2241683794>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAGCPUL62VQC7WLWGHUFH4DZNPIO5AVCNFSM6AAAAABIAKZOE2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDENBRGY4DGNZZGQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
